Sources/ArcGISToolkit/Components/Compass/Compass.swift

```diff
@@ -14,42 +14,34 @@
 import ArcGIS
 import SwiftUI
 
-/// A `Compass` (alias North arrow) shows where north is in a `MapView` or
-/// `SceneView`.
+/// A `Compass` (alias North arrow) shows where north is in a `MapView`.
 public struct Compass: View {
     /// The opacity of the compass.
     @State private var opacity: Double = .zero
     
-    /// The proxy to provide access to map view operations.
-    private var mapViewProxy: MapViewProxy?
-    
     /// A Boolean value indicating whether  the compass should automatically
     /// hide/show itself when the heading is `0`.
     private var autoHide: Bool = true
     
-    /// A Boolean value indicating whether the compass should hide based on the
-    ///  current heading and whether the compass automatically hides.
-    var shouldHide: Bool {
-        (heading.isZero || heading.isNaN) && autoHide
-    }
+    /// The heading of the compass in degrees.
+    private var heading: Double
+    
+    /// The proxy to provide access to map view operations.
+    private var mapViewProxy: MapViewProxy?
     
     /// The width and height of the compass.
     private var size: CGFloat = 44
     
-    /// The heading of the compass in degrees.
-    @Binding private var heading: Double
-    
-    /// Creates a compass with a binding to a heading based on compass
-    /// directions (0° indicates a direction toward true North, 90° indicates a
-    /// direction toward true East, etc.).
+    /// Creates a compass with a heading based on compass directions (0° indicates a direction
+    /// toward true North, 90° indicates a direction toward true East, etc.).
```

Comment:
Same question with this heading doc. Yes at a heading of 90° the needle is pointing eastward but the map would be oriented to the west.

Should this doc be written from the point of view of where the needle is pointing or how the map is oriented?

Reply:
I makes sense. again, this is talking about direction of rotation, which is opposite for `heading` than `rotation`.